website_analytics_matomo/views/templates.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <template id="layout" inherit_id="website.layout">
+        <xpath expr="//script[@id='tracking_code']" position="after">
+            <t
+                t-if="website and website.has_matomo_analytics and website.matomo_analytics_host"
+            >
+                <script type="text/javascript">
+                    var _paq = _paq || [];
+                    _paq.push(['trackPageView']);
+                    _paq.push(['enableLinkTracking']);
+                    (function() {
+                    var u="//<t t-raw="website.matomo_analytics_host" />/";
```

Comment:
I think this line should be this way:

```suggestion
                    var u="<t t-raw="website.matomo_analytics_host" />";
```

Reply:
Sorry @victor-champonnois I just noticed this [note in the field](https://github.com/coopiteasy/website/blob/14.0-mig-website_analytics_piwik/website_analytics_matomo/models/website.py#L19-L20) definition and [in the readme](https://github.com/coopiteasy/website/blob/14.0-mig-website_analytics_piwik/website_analytics_matomo/readme/CONFIGURE.rst) and I realized that what I wrote in my comment was wrong. The line I asked you to change was actually correct and with the last change it's not working properly anymore. My bad. So the last change should be reverted back as it was before. I apologize for that.

I'm thinking of a way to improve the UI for this configuration process, especially for distracted users like me. But that's not part for this PR. So for that I will probably open a new PR in a few days.